packages/types/src/mode.ts

```diff
@@ -66,6 +66,7 @@ export const modeConfigSchema = z.object({
 	name: z.string().min(1, "Name is required"),
 	roleDefinition: z.string().min(1, "Role definition is required"),
 	whenToUse: z.string().optional(),
+	description: z.string().optional(),
```

Comment:
How similar do you think this description will end up being to the "whenToUse"? There's something attractive to me about combining them (and also making the description of marketplace modes their "whenToUse"). 

For what it's worth we have a "slash command" feature where you can type `/` to see a list of modes, and we use the whenToUse for the description there. We should probably make that consistent with this either way.

Reply:
Looking farther down, I guess it's a lot shorter.